content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,225 @@
+
+* A﻿ set of notebooks that will be used by the student:
+* Containing instructions cells in markdown and run code cells leveraging the relevant kernel. If you are not familiar with Jupyter notebooks, a simple [101 workshop](https://developer.hpe.com/hackshack/workshop/25) is available in our Workshops-on-Demand 's catalog.
+* A student range for the workshop.
+
+A workshop should contain at least :
+
+* 0-ReadMeFirst.ipynb 
+* 1-WKSHP-LAB1.ipynb
+* 2-WKSHP-LAB2.ipynb
+* 3-WKSHP-Conclusion.ipynb
+* LICENCE.MD
+* A pictures folder (if any screenshot is required in lab instructions)
+* A README.md (0-ReadMeFirst.ipynb in md format)
+* A wod.yml
+
+To make the workshop compliant to our platform, Matt just needs to provide a final file that contains a set of metadata that will be used  for the workshop's integration into the infrastructure. this file is called **wod.yml**.
+
+Matt can leverage a simple **wod.yml** file containing them and that can be later parsed in order to feed the database with the relevant info. Quite handy, no? Moreover, the same script that will create the workshop entry in the ddatabase can also be sused to update it.
+
+Here is an example of such a file:
+
+```
+%YAML 1.1
+# Meta data for the GO101 Workshop to populate seeder
+---
+name: 'GO 101 - A simple introduction to Go Programming Language'
+description: 'Go, also called Golang or Go language, is an open source programming language that Google developed. Software developers use Go in an array of operating systems and frameworks to develop web applications, cloud and networking services, and other types of software. This workshop will drive you through the basics of this programming language.'
+active: true
+capacity: 20
+priority: 1
+range: [151-170]
+reset: false
+ldap: false
+location: 'fully qualified domain name of default JupyterHub server'
+replayId: 31
+varpass: false
+compile: false
+workshopImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/WOD-GO-101-A-simp-introduction-to-Go-programming-language.jpeg'
+badgeImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/go101-a-simple-introduction-to-go-programming-language.jpg'
+beta: false
+category: ['Open Source']
+duration: 4
+alternateLocation: ['fully qualified domain name of an alternate JupyterHub server']
+presenter: 'Matthew Doddler'
+role: 'FullStack developer'
+avatar: '/img/SpeakerImages/MattD.jpg'
+replayLink: 'https://hpe-developer-portal.s3.amazonaws.com/Workshops-on-Demand-Coming-Soon-Replay.mp4'
+```
+
+the following file will be used to update the **workshops table** in the database. Let's have a look at what a new entry could look like:
+
+![](/img/wod-db-go-1.png "GO 101 Workshop DB screenshot")
+
+
+
+![](/img/wod-db-go-2.png "GO 101 Workshop DB screenshot")
+
+As a contributor, Matt should be able to provide all the following details.
```

Comment:
Don't know if you need to add a dot at the end of each bullet item. Let's wait for Dale's review.